src/spaces/utilities.jl

```diff
@@ -153,6 +153,30 @@ end
 #######################################################################################
 # %% Walking
 #######################################################################################
+"""
+    normalize_position(pos, model::ABM{<:Union{AbstractGridSpace,ContinuousSpace}})
+
+Return the position `pos` normalized for the extents of the space of the given `model`.
+"""
+normalize_position(pos, model::ABM) = normalize_position(pos, model.space)
+
+function normalize_position(pos, space::ContinuousSpace{D,true}) where {D}
+    return mod.(pos, spacesize(space))
```

Comment:
I was using the `mod1` function, and here you use `mod`, what's the change for?

Reply:
`mod1(0.0, 2.0) == 2.0`, but continuous space doesn't include it's upper bound, and does include 0